src/SlideDeck.js

```diff
@@ -49,7 +44,7 @@ export class SlideDeck extends React.Component {
   static defaultProps = {
     slides: [],
     theme: defaultTheme,
-    components: {} ,
```

Comment:
Looks like prettier isn't running by default, but my editor settings picked it up and is now formatting code that others contributed before.